src/RoleDB.php

```diff
@@ -52,6 +90,31 @@ public function __construct()
         $this->roles = [];
     }
 
+    /**
+     * Add to the role database the 'roleinfo' associated with 'rolename'.
+     *
+     * @param $roleName
+     *     An object representing the role's name, conformant to 'ROLENAME_SCHEMA'
```

Comment:
Do you happen to know what `ROLENAME_SCHEMA` is a reference to @tedbow ? References to "conformant to *_SCHEMA" are commonplace in these comments but don't align with anything we currently have. In general I'm wondering how much we should rewrite these or provide definitions for them. I've just referenced sections of the TUF spec in some rewrites in KeyDB, but role names really aren't that complicated and am inclined to just delete that part.

Reply:
Yeah I think we can delete this. I create this issue https://github.com/php-tuf/php-tuf/issues/48
to talk about what schema validation we should have.

Reply:
Removed the references.